manifest.json

```diff
@@ -79,6 +84,13 @@
     },
     {
       "type": "string",
+      "format": "information",
+      "name": "reveal_trigger_prospector_info",
+      "title": "How does the trigger work?",
+      "headerTemplate": "The trigger let's you define when our connector automatically runs the Prospector workflow by defining the number of distinct users from the same IP address. As soon as this threshold is met, the connector will run the Prospect workflow if enabled."
```

Comment:
@unity and @sbellity, going forward we will just refer to the neutral style and use "the"...